README.md

```diff
@@ -6,7 +6,9 @@
 Project Peach Jam
 
 ## Prerequisites
-- postgresql
+- Python@3.10
+- libmagic
```

Comment:
I don't have libmagic installed and we don't install it in our Dockerfile -- perhaps a later python needed it but 3.10 doesn't?

Reply:
@actlikewill I left that part out because one of the migrations takes care of this step
`peachjam_ml/migrations/0001_pgvector.py`. Should I still include it in the README?

Reply:
@Jonathan-Ngondi We could still make a note of it in the docs and specifically mention that the database needs to have the pgvector plugin installed